repose-aggregator/components/services/datastore-service/datastore-service-impl/build.gradle

```diff
@@ -9,8 +9,6 @@ dependencies {
     compile "org.eclipse.jetty:jetty-servlet"
     compile "org.springframework:spring-beans"
     compile "net.sf.ehcache:ehcache-core"
-    compile "io.opentracing:opentracing-api"
```

Comment:
Oh wait. You probably still want this dependency. We don't like relying on transitive dependencies, and you do reference the `Tracer` class from this dependency directly.

I mostly just meant that we probably didn't need the `opentracing-util` dependency here since we aren't using the `GlobalTracer` or anything.